### DISCUSSION TITLE
Make drafts visible in development

### DISCUSSION THREAD

**Discussion by mpounsett** [getpelican/pelican#3083] [Q&A] [answered]
I'm trying to find a way to make draft articles visible in development... so from `make html` or `make devserver`.  

The only thing I can find that's even close is setting a custom variable in my theme that is set in `pelicanconf.py` and unset in `publishconf.py` to show a link to `drafts.html`.   Is there something better than that which I just haven't found?

**Comment by davidag** [comment, marked as answer]
[As per the docs](https://docs.getpelican.com/en/latest/content.html#publishing-drafts), draft posts are generated but hidden from any index. You can access them under the `drafts/` path.

I guess you probably don't want that, but to generate them as if they weren't drafts when you're in development.

An idea that *I think* could work is to just use the `published` status in posts and then have different default statuses for `DEFAULT_METADATA["status"]` for production and development:

* Development: `pelicanconf.py`
```python
DEFAULT_METADATA = {
    "status": "published"  # posts without a status tag will default to published
}
```
* Production: `publishconf.py`
```python
DEFAULT_METADATA = {
    "status": "draft"  # posts without a status tag will default to draft
}
```

**Reply by mpounsett** [reply]
Ah!  I'd been trying to think of something I could do with the two config files, but using `DEFAULT_METADATA` hadn't occurred to me.  This seems like it would work perfectly.  Thanks!
